editor/fileserver/editor_file_server.cpp

```diff
@@ -243,7 +245,9 @@ void EditorFileServer::_subthread_start(void *s) {
 				int read = cd->files[id]->get_buffer(buf.ptrw(), blocklen);
 				ERR_CONTINUE(read < 0);
 
-				DEBUG_PRINT("GET BLOCK - offset: " + itos(offset) + ", blocklen: " + itos(blocklen));
+				if (OS::get_singleton()->is_stdout_verbose()) {
+					print_line("GET BLOCK - offset: " + itos(offset) + ", blocklen: " + itos(blocklen));
```

Comment:
You can use `print_verbose` to replace this exact construct.

Reply:
Oh, didn't know we had this function.